File: Web/src/app/models/bus.ts
import {Stop} from "./stop";

export interface Bus {
  id: string
  number: string
  start: string
  end: string
  startLat: number
  startLng: number
  endLat: number
  endLng: number
  type: string
}

export interface FullBus {
  bus: Bus
  stops: Stop[]
}

File: Api/api/src/main/kotlin/com/likbus/app/buses/models/Bus.kt
package com.likbus.app.buses.models

import kotlinx.serialization.Serializable

@Serializable
data class Bus(
    val id: String,
    val number: String,
    val start: String,
    val end: String,
    val startLng: Float,
    val startLat: Float,
    val endLng: Float,
    val endLat: Float,
    val type: String
)
